sync/vm/client.go

```diff
@@ -34,6 +34,7 @@ const (
 	ParentsToFetch = 256
 
 	atomicStateSyncOperationName = "Atomic State Sync"
+	blockStateSyncOperationName  = "Block State Sync"
```

Comment:
The blocks really aren't "state".
```suggestion
	blockSyncOperationName       = "Block Sync"
```